lib/load_config.js

```diff
@@ -15,11 +16,17 @@ var yaml = require('js-yaml'),
  * @throws {Error} if the file cannot be read.
  */
 function loadConfig(filePath) {
+  var ext = _.last(filePath.split('.'));
```

Comment:
we have the `path` module available - can this use [path.extname](https://nodejs.org/api/path.html#path_path_extname_path) instead, to more directly use a method designed for the purpose of getting extensions?